group_vars/all/public.yml

```diff
@@ -0,0 +1,11 @@
+tarsnap_version: 1.0.37
+tarsnap_cache: /var/tarsnap/cache
+tarsnap_cron_hour: "*"
+tarsnap_tarsnapper_conf: "{{ playbook_dir }}/opencraft-roles/roles/tarsnap/files/conf/tarsnapper.{{ inventory_hostname }}.conf"
+
+OPS_EMAIL: 'jacek@opencraft.com'
```

Comment:
This should also be set publicly to a non-functional email (ops@example.com), and overriden in the private vars.